rskj-core/src/test/java/co/rsk/core/BlockHeaderTest.java

```diff
@@ -105,7 +104,7 @@ void getHashForMergedMiningWithForkDetectionDataAndIncludedOff() {
 
     @Test
     void getEncodedWithUmmRootWithMergedMiningFields() {
-        byte[] ummRoot = TestUtils.randomBytes(20);
+        byte[] ummRoot = new byte[]{0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19};
```

Comment:
I didn't use the TestUtil object here, to follow the same behaviour as other tests into this class where the arrays are being initialised manually.

Reply:
I think we could take this chance to unify it by using `TestUtils`, WDYT?

Reply:
It is not strictly necessary but I could do it.